virtcontainers/kata_agent.go

```diff
@@ -1082,7 +1089,18 @@ func (k *kataAgent) handleBlockVolumes(c *Container) []*grpc.Storage {
 		}
 
 		vol.MountPoint = m.Destination
-		vol.Fstype = "bind"
+		ac, _ := sandbox.config.AgentConfig.(KataAgentConfig)
```

Comment:
Not sure if its necessary since it doesn't look like there are other Agent config types, but do you want to check whether the type assertion is true so that it doesn't panic if for some reason its not?